app/Repositories/CacheRepository.php

```diff
@@ -26,16 +34,21 @@ public function retrieve($key)
      */
     public function retrieveMany(array $keys)
     {
+        $idsWithPrefix = [];
+
+        foreach ($keys as $key => $id) {
+            array_push($idsWithPrefix, $this->setPrefix($id));
```

Comment:
Can you rename `setPrefix` to `applyPrefix` or something similar, and make it a protected method? At first glance, `setPrefix` reads as a setter method.